pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1200,7 +1227,6 @@ func getDefaultQueue(append bool, defaultJmsQueue string, jmsQueue string) strin
 func setPasswords(spec *api.KieAppSpec, isTrialEnv bool) {
 	passwords := []*string{
 		&spec.CommonConfig.KeyStorePassword,
-		&spec.CommonConfig.AdminPassword,
```

Comment:
why remove it from here?

Reply:
Because now we handle the password earlier in the flow and in one single point newr to the handle point of the secret, this check is redundant